torch/_inductor/codegen/cpp_utils.py

```diff
@@ -681,3 +681,170 @@ def get_gemm_template_output_and_compute_dtype(input_dtype):
         return (torch.int32, torch.int32)
     else:
         return (torch.float32, torch.float32)
+
+
+# Factored out epilogue creators here
```

Comment:
Do we need a comment highlighting "factored out" here?

Reply:
Will remove, thanks!